src/containers/Registration.tsx

```diff
@@ -0,0 +1,209 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import React, { useState } from "react";
+import { Modal, Button, Form, FloatingLabel } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.css";
+import { useForm } from "react-hook-form";
+import styled from "styled-components";
+import * as yup from "yup";
+
+const WrapperDiv = styled.div`
+  > * {
+    font-family: TwitterChirp, -apple-system, BlinkMacSystemFont, "Segoe UI",
+      Roboto, Helvetica, Arial, sans-serif;
+  }
+`;
+
+const StyledModalTitle = styled(Modal.Title)`
+  text-align: center;
+  font-weight: 700;
+  font-size: 2rem;
+`;
+
+const StyledFormControl = styled(Form.Control)`
+  margin-top: 1rem;
+  padding: 2rem 0.75rem 2rem 1.7rem !important;
+  height: 5.5rem !important;
+  font-size: 1.5rem;
+`;
+
+const StyledFloatingLabel = styled(FloatingLabel)`
+  font-size: 1.5rem;
+  label {
+    padding: 2rem 1.7rem;
+  }
+  .form-floating > .form-control:focus ~ label,
+  .form-control:not(:placeholder-shown) ~ label,
+  .form-control:focus ~ label {
+    transform: scale(0.8) translateY(-1.5rem) translateX(0);
+    opacity: 0.65 !important;
+  }
+`;
+
+const StyledCounterLabel = styled.label`
+  text-align: right;
+  opacity: 0;
+  padding-right: 0;
+  margin-right: 0;
+  min-width: 124%;
+  transform-origin: 0 0;
+`;
+
+interface MyForm {
+  userName: string;
+  email: string;
+  dateOfBirth: Date;
```

Comment:
not sure if it's proper type
common practice is to use ISO string data format to save dates
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

Reply:
ISO should be a string type

Reply:
Change date type to string